groovy/postInit/gameplay/Invasions.groovy

```diff
@@ -3,21 +3,35 @@ import net.minecraft.entity.monster.EntityZombie;
 import net.minecraft.init.MobEffects;
 import net.minecraft.potion.PotionEffect;
 import techguns.entities.npcs.Bandit;
+import flaxbeard.cyberware.common.entity.EntityCyberZombie;
 
-new MobHordeEvent((player) -> {return new EntityZombie(player.world);}, 5, 10, "zombie_medium")
+new MobHordeEvent((player) -> {EntityZombie zombie = new EntityZombie(player.world);
+			        zombie.addPotionEffect(new PotionEffect(MobEffects.SPEED, 999999, 1));
+				return zombie;}, 5, 10, "zombie_medium")
 	.setAdvancementUnlock(new ResourceLocation("gregtech:steam/4_bronze_boiler"))
 	.setNightOnly(true)
 	.setTimer(144000, 216000)		// 2 - 3 hours
+	.setCanUsePods(false)
 
 new MobHordeEvent((player) -> {EntityZombie zombie = new EntityZombie(player.world); 
 			        zombie.addPotionEffect(new PotionEffect(MobEffects.REGENERATION, 999999, 1));
 			        zombie.addPotionEffect(new PotionEffect(MobEffects.STRENGTH, 999999, 1));
-			        zombie.addPotionEffect(new PotionEffect(MobEffects.SPEED, 999999));
+			        zombie.addPotionEffect(new PotionEffect(MobEffects.SPEED, 999999, 1));
 				return zombie;}, 6, 12, "zombie_hard")
 	.setAdvancementUnlock(new ResourceLocation("gregtech:steam/16_steel_boiler"))
 	.setNightOnly(true)
 	.setTimer(144000, 216000)		// 2 - 3 hours
+	.setCanUsePods(false)
 
 new MobHordeEvent((player) -> {Bandit bandit = new Bandit(player.world); bandit.addRandomArmor(0); return bandit;}, 2, 4, "bandit_medium")
 	.setAdvancementUnlock(new ResourceLocation("gregtech:low_voltage/23_lv_assembler"))
 	.setTimer(72000, 216000)	        // 1 - 3 hours
+
+new MobHordeEvent((player) -> {EntityCyberZombie zombie = new EntityCyberZombie(player.world);
```

Comment:
Don't add cyberzombies to any of the Earth dimensions

Reply:
Removed the invasion then. By the time extra planets appear another one can be quickly drafted up.

Reply:
> Don't add cyberzombies to any of the Earth dimensions

should we add smth else?